### PR TITLE
Bugfix in average_pool2d, missing divide by the sum of strides.

### PR DESCRIPTION
**Pull request by david-berthelot**

### CODE REVIEW

File: tests/functional_pooling.py [hunk 1]
```diff
@@ -27,6 +27,18 @@ def shaparange(s):
 
 
 class TestPooling(unittest.TestCase):
+    def test_average_pooling2d(self):
+        """Test batch_to_space2d and space_to_batch2d."""
```
Comment (kihyuks) [inline tests/functional_pooling.py:31]: Change docstring into Test average pooling 2d

File: tests/functional_pooling.py [hunk 2]
```diff
@@ -27,6 +27,18 @@ def shaparange(s):
...
+        x = shaparange((2, 3, 10, 30))
+        y = objax.functional.average_pool_2d(x, size=5)
+        z = x.reshape((2, 3, 2, 5, 6, 5)).mean((-3, -1))
+        self.assertEqual(y.tolist(), z.tolist())
+        y = objax.functional.average_pool_2d(x, size=5, strides=1)
```
Comment (kihyuks) [inline tests/functional_pooling.py:36]: Can we have a test with a small tensor to check the output value of average_pool_2d analytically when the size and strides are different?

**Review by kihyuks — CHANGES_REQUESTED** [review]

File: tests/functional_pooling.py (hunk 2 above)
Comment (david-berthelot) [reply tests/functional_pooling.py:35]: Done

File: tests/functional_pooling.py (hunk 1 above)
Comment (david-berthelot) [reply tests/functional_pooling.py]: Actually I removed the docstring, it's redundant with the test name and provided no useful information.

File: tests/functional_pooling.py (hunk 2 above)
Comment (david-berthelot) [reply tests/functional_pooling.py:35]: Done

**Pull request merged**